LibreNMS/Alert/Transport/Twilio.php

```diff
@@ -0,0 +1,113 @@
+<?php
+/*
+ * This program is free software: you can redistribute it and/or modify it
+ * under the terms of the GNU General Public License as published by the
+ * Free Software Foundation, either version 3 of the License, or (at your
+ * option) any later version.  Please see LICENSE.txt at the top level of
+ * the source code distribution for details.
+*/
+/**
+ * Twilio API Transport
+ * @author Andy Rosen <arosen@arosen.net>
+ * @license GPL
+ * @package LibreNMS
+ * @subpackage Alerts
+ */
+namespace LibreNMS\Alert\Transport;
+
+use LibreNMS\Alert\Transport;
+
+class Twilio extends Transport
+{
+    public function deliverAlert($obj, $opts)
+    {
+        if (empty($this->config)) {
```

Comment:
This is only needed for legacy transports, as this is new then it won't have legacy config. Remove this check + the `devliverAlertOld()` function